nameko/standalone/events.py

```diff
@@ -31,6 +34,11 @@ def event_dispatcher(nameko_config, **kwargs):
     def dispatch(service_name, event_type, event_data):
         conn = Connection(nameko_config[AMQP_URI_CONFIG_KEY])
 
+        if KOMBU_SERIALIZER_CONFIG_KEY in nameko_config.keys():
+            serializer = nameko_config[KOMBU_SERIALIZER_CONFIG_KEY]
+        else:
+            serializer = DEFAULT_KOMBU_SERIALIZER
```

Comment:
Nicer to use the ``get`` method with a default:

``` python
serializer = nameko_config.get(SERIALIZER_CONFIG_KEY, DEFAULT_SERIALIZER)
```

Not sure why we do ``config.get(MAX_WORKERS_CONFIG_KEY) or DEFAULT_MAX_WORKERS)`` further up. Maybe you can fix that too...